### DISCUSSION TITLE
程序进去运行没有反应，然后就是报错

### DISCUSSION THREAD

**Discussion by VIOLETextra** [mikumifa/biliTickerBuy#416] [提问专区]
点开程序，进去之后，先是有一段，终端是空白的，之后就突然出现一段字符，然后程序就突然关闭了，只能截下这张图
![屏幕截图 2024-08-27 170621](https://github.com/user-attachments/assets/59ce08a5-59a3-418f-b8a0-88d3199fcca4)

**Comment by WittF** [comment]
打包缺少依赖了，等新版本发布

**Comment by mikumifa** [comment]
[Release v2.8.8-patch.1](https://github.com/mikumifa/biliTickerBuy/releases/tag/v2.8.8-patch.1)
